app/main.ts

```diff
@@ -23,10 +25,23 @@ bootstrap(AppComponent, [
         provide: LocationStrategy,
         useClass: HashLocationStrategy
     },
+    // TODO: (pay attention) Here is declared global services and thay must declares only here, don't add it to
```

Comment:
It's very important please remember that moment if you will use angular 2 DI in the future.